dart/looker_sdk/lib/src/rtl/api_types.dart

```diff
@@ -0,0 +1,18 @@
+class DelimList<T> {
```

Comment:
I like that `DelimList` is shorter than `DelimArray`. Renaming in other SDKs is probably not worth the breakage.